__tests__/app.test.js

```diff
@@ -0,0 +1,30 @@
+const app = require("../app/app.js");
+const db = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+const request = require("supertest");
+const data = require("../db/data/test-data");
+
+beforeEach(() => {
+    return seed(data);
+  });
+  afterAll(() => {
+    db.end();
+  });
+
+describe("GET /api/topics", () => {
+  it("return an array of objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((res) => {
+        const topics = res.body.topics;
+        expect(topics.length).toBe(3);
+        topics.forEach((topic) => {
+          expect(topic).toMatchObject({
+            description: expect.any(String),
+            slug: expect.any(String),
+          });
+        });
+      });
```

Comment:
Is there any sad path to deal with? What if the user enter the wrong url? e.g. /api/topiczzz